app/components/Sidebar/components/CollectionLink.js

```diff
@@ -136,6 +194,11 @@ function CollectionLink({
   );
 }
 
+const Draggable = styled("div")`
+  opacity: ${(props) => (props.$isDragging || props.$isMoving ? 0.5 : 1)};
+  pointer-events: ${(props) => (props.$isMoving ? "none" : "all")};
```

Comment:
this is a small nit — would `"auto"` be a better default value than `"all"` here

Reply:
Done, Thanks Nan!